ui/raidboss/data/04-sb/alliance/orbonne_monastery.js

```diff
@@ -855,22 +824,19 @@
         'Halidom': 'faux sanctuaire',
         'Harpy': 'harpie',
         'I see it now': 'À vous, maintenant',
-        'Icewolf': 'grêlon de glace',
         'Iron Construct': 'bâtisseur de fer',
         'Mustadio': 'Mustadio',
         'Ramza': 'Ramza',
-        'Ruination': 'Ruinance',
-        'Shield Knight': 'chevalier au bouclier',
         'Sword Knight': 'chevalier à l\'épée',
-        'The Crystalline Gaol': 'la Geôle cristalline',
+        'The Crystalline Gaol': 'Geôle cristalline',
```

Comment:
Why are you removing the `la` here? I think this article is needed to make the regex `Fermeture d(?:e|u|es) la Geôle cristalline dans` is it really `Fermeture d(?:e|u|es) Geôle cristalline dans` in game?